File: Codesignal/Eric/72.AddBorder.py
from pprint import pprint

def solution(picture):
    result = []
    result.append('*' * (len(picture[0]) + 2))
    for row in picture:
        result.append('*' + row + '*')
    result.append('*' * (len(picture[0]) + 2))
    return result

def solution2(picture):
    stars = ['*' * (len(picture[0]) + 2)] 
    return stars + ['*' + row + '*' for row in picture] + stars

pprint(solution2(["abc", "ded"]))

File: Codesignal/Eric/72.AddBorder.js
function solution(pic) {
    const stars = '*'.repeat(pic[0].length + 2);
    let list = [];
    list.push(stars);
    pic.map (x => list.push('*' + x + '*'));
    list.push(stars);
    return list
}


function solution2(pic) {
    return [x='*'.repeat(pic[0].length + 2), ...pic.map(x => '*' + x + '*'), x]
}

console.log(solution2(["abc", "ded"]))